spec/integration/api/conversations_controller_spec.rb

```diff
@@ -64,13 +115,15 @@
     it "returns all the user conversations" do
       get api_v1_conversations_path, params: {access_token: access_token}
       expect(response.status).to eq 200
-      expect(JSON.parse(response.body).length).to eq 3
+      returned_convos = JSON.parse(response.body)
```

Comment:
`returned_conversation` would be clearer.

Reply:
Same as above can get rid of the abbreviation

Reply:
Yep please